lib/tslint/5.15.x/tslint.json

```diff
@@ -0,0 +1,191 @@
+    "no-redundant-jsdoc": true,
+    "no-reference-import": true,
+    "no-unnecessary-callback-wrapper": false,
+    "object-literal-shorthand": false,
+    "one-variable-per-declaration": true,
+    "switch-final-break": false,
+    "variable-name": [
+      true,
+      "check-format",
+      "require-const-for-all-caps",
+      "allow-leading-underscore",
+      "allow-trailing-underscore",
+      "ban-keywords"
+    ],
+    "whitespace": [
+      false,
+      "check-branch",
+      "check-decl",
+      "check-operator",
+      "check-module",
+      "check-separator",
+      "check-rest-spread",
+      "check-type",
+      "check-typecast",
+      "check-type-operator",
+      "check-preblock"
+    ],
+    "prefer-const": [
+      true,
+      {
+        "destructuring": "all"
+      }
+    ],
+    "no-magic-numbers": false,
+    "import-blacklist": [true, "@angular/material", "lodash", "rxjs/internal", "rxjs/index", "rxjs/Rx", "lodash-es"],
+    "ordered-imports": false,
+    "max-classes-per-file": false, // [true, 1],
+    "prefer-for-of": true,
+    "array-type": false, // [true, "array"]
+    "arrow-return-shorthand": true,
+    "binary-expression-operand-order": false,
+    "no-unnecessary-initializer": true,
+    "no-misused-new": true,
+    "prefer-method-signature": false,
+    "prefer-switch": [
+      true,
+      {
+        "min-cases": 3
+      }
+    ],
+    "prefer-function-over-method": false,
+    "prefer-while": true,
+    "no-import-side-effect": [
+      true,
+      {
+        "ignore-module": "(reflect-metadata|core-js|ngrx-store|angular|moment|polyfills|vendor|typings|\\.css|\\.pcss)"
+      }
+    ],
+    "no-non-null-assertion": true,
+    "no-parameter-reassignment": false,
+    "adjacent-overload-signatures": true,
+    "ban-comma-operator": false,
+    "ban-types": false,
+    "no-duplicate-super": true,
+    "no-duplicate-switch-case": true,
+    "newline-before-return": false,
+    "newline-per-chained-call": false,
+    "file-name-casing": [true, { "public_api.ts": "snake-case", ".*": "kebab-case" }],
+    "ban-ts-ignore": true,
+    "function-constructor": true,
+    "unnecessary-constructor": true,
+    "no-tautology-expression": true,
+    "static-this": true,
+    "unnecessary-else": true,
+
+    // TSlint rules that require type info
+    "no-inferred-empty-object-type": true,
+    "no-void-expression": [true, "ignore-arrow-function-shorthand"],
+    "no-boolean-literal-compare": true,
+    "no-unnecessary-qualifier": true,
+    "no-floating-promises": true,
+    "match-default-export-name": true,
+    "return-undefined": true,
+    "deprecation": true,
+    "use-default-type-parameter": true,
+    "await-promise": true,
+    "no-for-in-array": true,
+    "no-unbound-method": [true, { "ignore-static": true, "whitelist": ["expect"], "allow-typeof": true, "allow-delete": true }],
+    "no-unnecessary-type-assertion": true,
+    "restrict-plus-operands": true,
+    "unnecessary-bind": true
```

Comment:
Could you maybe add the rule `"no-null-undefined-union": true` ? 
I enabled it in Stark and it does not break anything. 
According to the [documentation](https://palantir.github.io/tslint/rules/no-null-undefined-union/), it's interesting to enable this because it allows the `triple-equals` rule to exist without exceptions.

Reply:
True, I'll add it ;)